include/wx/bmpbndl.h

```diff
@@ -76,6 +76,9 @@ class WXDLLIMPEXP_CORE wxBitmapBundle
     // This overload currently makes a copy of the data.
     static wxBitmapBundle FromSVG(const char* data, const wxSize& sizeDef);
 
+    // This overload can work on non 0 terminted data
```

Comment:
```suggestion
    // This overload can work on non 0 terminated data
```

I wonder why this hasn't been caught by codespell...